src/libs/OptionsListUtils.js

```diff
@@ -937,8 +932,14 @@ function getTagsOptions(tags) {
  * @param {Number} maxRecentReportsToShow
  * @returns {Array<Object>}
  */
-function getTagListSections(tags, recentlyUsedTags, selectedOptions, searchInputValue, maxRecentReportsToShow) {
+function getTagListSections(rawTags, recentlyUsedTags, selectedOptions, searchInputValue, maxRecentReportsToShow) {
     const tagSections = [];
+    const tags = _.map(rawTags, (tag) => {
+        // This is to remove unnecessary escaping backslash in tag name sent from backend.
+        const tagName = tag.name && tag.name.replace(/\\{1,2}:/g, ':');
+
+        return {...tag, name: tagName};
```

Comment:
I think we need to update `searchText`,` tooltipText` as well, no?

Reply:
Nope. searchText and tooltipText are generated in here
https://github.com/Expensify/App/blob/48a2b46cb5c5c059c4a78301f896a3f5283814d0/src/libs/OptionsListUtils.js#L912-L920
all from tag.name that's why we are removing the slash here so that all the logic below will be correct :+1:

Reply:
Ah I see. Thanks for clarifying!